setup/install/providers/aws/aws-ec2.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: single
+title:  "Amazon EC2"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In [AWS](https://aws.amazon.com/){:target="\_blank"}, an [__Account__](/concepts/providers/#accounts)
+maps to a credential able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/){:target="\_blank"}.
+
+## Option-1 : Use AWS Console to configure AWS
+
+Use this option to deploy Spinnaker, if you are familar with deployment using [AWS Console](https://console.aws.amazon.com/) .
+
+### Managing Account
+1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
+2. Download [the template](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managing.yaml) locally to your workstation.
+    
+    2.a (Optional). Add Managing account as shown [here](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managing.yaml#L104)
+3. Creating the CloudFormation Stack
+    * __Create Stack__ > __Upload a template to Amazon S3__ > __Browse to template you downloaded in Step-2 above__ > __Next__  
+    * Enter __Stack Name__ as spinnaker-__managing__-infrastructure-setup and follow the prompts on screen to create the stack
+4. Once the stack is select the stack you created in Step-3 > Outputs and note the values. You will need these values for subsequent configurations.
+
+
+### In each of the Managing Account
+
+> These steps need to be carried out for the managing account as well.
+
+1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
+2. Download [the template](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managed.yaml) locally to your workstation.
+3. Creating the CloudFormation Stack
+    * __Create Stack__ > __Upload a template to Amazon S3__ > __Browse to template you downloaded in Step-2 above__ > __Next__  
+    * Enter __Stack Name__ as spinnaker-__managed__-infrastructure-setup and follow the prompts on screen to create the stack
```

Comment:
The markdown doesn't work on this __managed__ part. 
<img width="885" alt="screen shot 2018-06-19 at 12 37 48 pm" src="https://user-images.githubusercontent.com/8595511/41620129-9a2d47bc-73bd-11e8-9dfe-d7377ac5309d.png">